src/stream/src/executor/lookup/impl_.rs

```diff
@@ -381,14 +382,14 @@ impl<S: StateStore> LookupExecutor<S> {
         {
             let all_data_iter = self
                 .arrangement
-                .state_table
-                .iter_with_pk_prefix(&lookup_row, lookup_epoch)
-                .await?
-                .fuse();
+                .storage_table
+                .streaming_iter_with_pk_bounds(lookup_epoch, &lookup_row, ..)
+                .await?;
             pin_mut!(all_data_iter);
             while let Some(inner) = all_data_iter.next().await {
                 let ret = inner?;
-                all_rows.push(ret.into_owned());
+                // Only need value (include storage pk).
+                all_rows.push(ret.1.clone());
```

Comment:
Here may be a problem. If we adopt row-based encoding storage table + dedup pk in future, the return `Row` may not include storage pk. Stoarge pk is `Vec<u8>` and needs further desr. cc @BugenZhao @wcy-fdu

Reply:
I believe dedup-pk deserializer will help us to fill the deduplicated columns in `Row`?

Reply:
Hmmm yes, this is the question that should be answered when we dedup the scan results. Marked as Resolve now.